__tests__/utilities/max.js

```diff
@@ -1,4 +1,4 @@
-const max = require('../../src/utilities/max');
+const { max } = require('../../src/utilities/max');
```

Comment:
I think we should use imports instead of require.

Reply:
Which files would you like me to change? Just the one?

Changing some (such as `neural-networks.js`) requires a lot of files in the codebase to instead use the `.default`

Reply:
I have updated imports except in `__tests__`